tools/malwarejail/env/wscript.js

```diff
@@ -2201,3 +2183,62 @@ ga = function () {
 XPathResult = function () {
     return XPathResult;
 }
+
+// These constructors override the original class via aliasing.
+Blob = function() {
+    const _id = ++_object_id;
```

Comment:
Every usage of _object_id is post-incrementing it, I think you should do the same here, instead of pre-incrementing it.